src/Promitor.Tests.Unit/Serialization/v1/Core/V1DeserializerTests.cs

```diff
@@ -34,16 +34,18 @@ public V1DeserializerTests()
         }
 
         [Fact]
-        public void Deserialize_NoVersionSpecified_ThrowsException()
```

Comment:
What happens if these are not specified and you start up the app? Does it still block it from proceeding?

Reply:
Yeah - in all cases it blocks the app from starting properly. The version is actually a bit of an odd one - it won't even get as far as the deserializer if the version is missing because we don't know what version of the config to deserialize!

Anyhow, here's examples of the output in each case.

- No version: https://gist.github.com/adamconnelly/99ec2264cf8e796a54f40d0bd6b3df22
- Metrics missing: https://gist.github.com/adamconnelly/1c402535d4eca8f07fc3d9e7662af6b6
- Defaults missing: https://gist.github.com/adamconnelly/c8016c675e81190cd3d81e8b9479a292
- Azure metadata missing: https://gist.github.com/adamconnelly/d14d422bc8ee3cbf9b38cda3482308be

Reply:
Awesome, thanks!